include/zephyr/net/wifi_mgmt.h

```diff
@@ -53,6 +53,8 @@ extern "C" {
 enum net_request_wifi_cmd {
 	/** Get Wi-Fi driver and Firmware version */
 	NET_REQUEST_WIFI_CMD_VERSION = 1,
+	/** Set or get MAC Address */
```

Comment:
This is already available as part of net-mgmt commands/APIs, so, no need to for a new one.

Reply:
Didn't get your point

Reply:
Please see https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/lib/shell/iface.c#L238 `net iface <idx>` shows this info.

Reply:
The API is for get/set both.

Reply:
We can do that too `set_mac`, MAC address is common across all network interfaces not just Wi-Fi, so, it belongs in the `net shell`.

```
$ net iface 1 help
iface: wrong parameter count
iface - Print information about network interfaces.
Subcommands:
  up       : 'net iface up <index>' takes network interface up.
  down     : 'net iface down <index>' takes network interface down.
  show     : 'net iface <index>' shows network interface information.
  set_mac  : 'net iface set_mac <index> <MAC>' sets MAC address for the network
            interface.
```

Reply:
We can use that.